ci-operator/jobs/openshift/knative-serving/openshift-knative-serving-release-next-presubmits.yaml

```diff
@@ -4,17 +4,17 @@ presubmits:
     always_run: true
     branches:
     - release-next
-    context: ci/prow/4.3-e2e-aws-ocp-43
+    context: ci/prow/4.2-e2e-aws-ocp-42
```

Comment:
Are all the 4.3 -> 4.2 changes in the generated jobs expected? It looks like a corresponding 4.2 -> 4.3 change for each as well, perhaps? So just ordering issues in the diff?

Reply:
yeah seems like ordering. I ditched the `__variant` and "unnamed" files and renamed them to be `__4.x` respectively.